arithmetization/src/main/java/net/consensys/linea/zktracer/module/constants/GlobalConstants.java

```diff
@@ -25,18 +25,19 @@
  */
 public class GlobalConstants {
   public static final int BLOCKHASH_MAX_HISTORY = 0x100;
+  public static final int CREATE2_SHIFT = 0xff;
```

Comment:
@powerslider  I'm wondering if these should be part of this PR. Does it mean that constants.lisp was changed without generating GlobalConstants.java ?

Reply:
@ahamlat It means that it was not recently regenerated, so that is fine.